### DISCUSSION TITLE
Mastodon with S3 bucket

### DISCUSSION THREAD

**Discussion by polkonic** [mastodon/mastodon#27942] [General]
hello, 

does anyone use Google Cloud Storage (Bucket) as s3 provider.

could you share your .env config setting please?

i can work with amazon s3 and Cloudfront. it works great as expected.

but i tried with Wasabi, Digitalocean, Google Cloud. it seems none of them working, My CORS setting is fine. 

Anyone please, help me setup with Google storage, been searching days now. 

thanks. 

I am using latest 3.1.4 version.

**Comment by angristan** [comment]
If that can help 

- https://stanislas.blog/2018/05/moving-mastodon-media-files-to-wasabi-object-storage/
- https://stanislas.blog/2019/11/object-storage-migrate-wasabi-to-scaleway-rclone/

Both include the env config used.
